tests/shared/drive-constraints.spec.js

```diff
@@ -303,52 +303,219 @@ describe('Shared: DriveConstraints', function() {
 
   describe('.isDriveLargeEnough()', function() {
 
-    it('should return true if the drive size is greater than the image size', function() {
-      const result = constraints.isDriveLargeEnough({
+    beforeEach(function() {
+      this.drive = {
         device: '/dev/disk1',
         name: 'USB Drive',
-        size: 1000000001,
+        size: 1000000000,
         protected: false
-      }, {
-        path: 'rpi.img',
-        size: 1000000000
+      };
+    });
+
+    describe('given the final image size estimation flag is false', function() {
+
+      describe('given the original size is less than the drive size', function() {
+
+        beforeEach(function() {
+          this.image = {
+            path: path.join(__dirname, 'rpi.img'),
```

Comment:
There's lots of other image relative-only `path` properties scattered throughout the test scripts. If you like I could make the change you've done here to all of them, in a separate PR after you've merged this one?